src/Uno.UWP/Data/Pdf/PdfPageRenderOptions.Android.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using Windows.Foundation;
+using Windows.UI;
+
+namespace Windows.Data.Pdf;
+
+public partial class PdfPageRenderOptions
+{
+	public Rect SourceRect { get; set; }
+
+	public bool IsIgnoringHighContrast { get; set; }
```

Comment:
Consider verifying whether the default is `true` or `false` on UWP. If it's `false, we're good. If it's true, we need to set it.

Reply:
Is this property actually used? If not, I think it should remain as NotImplemented